### PR TITLE
task 4

### PR DESCRIPTION
**Pull request by Zoba1**
i am done with my task 4 please review
@chidipatrick
@promise-J

### CODE REVIEW

**Review by promise-J — APPROVED** [review]
@Zoba1 , please add the tab logo for the page.

File: html/index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            <h1 id="top">Shop all kinds of PET</h1>
+            <nav>
+                <a href="#">Home</a>
+                <a href="#">About</a>
+                <a href="#">contactus</a>
+            </nav>
+        </header>
+        <section>
+            <p>
+                <a name="section"></a>
```
Comment (ChidiPatrick) [inline html/index.html]: This anchor element has no content. Add the content, and the "href" attribute and value.

**Review by ChidiPatrick — CHANGES_REQUESTED** [review]
Fix the anchor tag issue I commented on down there

**Review by ChidiPatrick — APPROVED** [review]
Great job, keep it up

**Pull request merged**